koku/api/report/test/utils.py

```diff
@@ -188,18 +202,25 @@ def load_aws_data(self, customer, static_data_file, account_id=None, role_arn=No
             run(nise_provider_type.lower(), options)
 
             report_path = self.build_report_path(provider_type, bill_date, base_path)
-            for report in os.scandir(report_path):
-                if os.path.isdir(report):
-                    for report in [f.path for f in os.scandir(f"{report_path}/{report.name}")]:
-                        if os.path.isdir(report):
-                            continue
-                        elif "manifest" in report.lower():
-                            continue
-                        self.process_report(report, "GZIP", provider_type, provider, manifest)
+            try:
```

Comment:
I'm not sure we need to catch exceptions here. The koku-test-runner has broad exception handling, so anything that gets raised here is caught there.

Reply:
That might be true, but I don't think that's desirable behavior.

The main reason for catching the IOError here is to log the error and have something meaningful in the console output if there's a failure processing report data.